.gitignore

```diff
@@ -3,6 +3,7 @@
 .bundle
 .config
 .yardoc
+.ruby-version
```

Comment:
just not sure if/why ignoring ruby-version is a good idea. A quick check in some other repos of ours shows we usually don't ignore it...

Reply:
I'll remove this

Reply:
Done!